nion/swift/ExportDialog.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+import enum
```

Comment:
This should be before `import gettext` below. The imports are all in the same section and alphabetized.

Reply:
We should also remove the unused `import functools`. I had deleted that in my local copy, but if it were to remain, `import enum` should go (alphabetically) before `import functools`.